lib/utils/process-jobs.ts

```diff
@@ -253,7 +253,7 @@ export const processJobs = async function(this: Agenda, extraJob: Job) {
           // This means a job has "expired", as in it has not been "touched" within the lockoutTime
           // Remove from local lock
           // NOTE: Shouldn't we update the 'lockedAt' value in MongoDB so it can be picked up on restart?
-          if (job.attrs.lockedAt < lockDeadline) {
+          if (!job.attrs.lockedAt || job.attrs.lockedAt < lockDeadline) {
```

Comment:
same for this check, shouldn it be:
`          if (job.attrs.lockedAt && job.attrs.lockedAt < lockDeadline) {
`